src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -135,7 +134,71 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * BkBasic produces headers with addresses without slashes.
+     *
+     * @throws IOException If some problem inside
+     */
```

Comment:
@dmzaytsev it will be more correct to define @todo at class level, because createMockSocket used twice. Not only in my test